### ISSUE TITLE
Firebase error when changing scene title 

### ISSUE THREAD

**Issue by Algorush** [3DStreet/3dstreet#532]
if I try to click on the scene title field, then after entering a new name for the street in a prompt window, an error appears:
```
Error with update title Error: FirebaseError: [code=invalid-argument]: Function doc() cannot be called with an empty path.
    at scene.js:87:11
```

**Comment by kfarr** [comment]
Yes, this occurs if the file has not been saved yet. somewhere there should be the check to see if a doc exists, if not then don't try to update the title

**Comment by Algorush** [comment]
I think this error appears because the sceneTitle update cannot happen without sceneId. Now the sceneId is generated only when the scene is saved to the cloud. That is, if the scene is loaded from Streetmix or from a JSON file from disk, then the sceneId is missing. And therefore, saving sceneTitle in Firebase cannot occur and this error appears.

**Comment by kfarr** [comment]
closed by #619
